query_optimizer/resolver/Resolver.cpp

```diff
@@ -2794,6 +3150,10 @@ void Resolver::rewriteIfOrdinalReference(
   }
 }
 
+std::string Resolver::GenerateWindowAggregateAttributeAlias(int index) {
+  return std::string("$window_aggregate").append(std::to_string(index));
```

Comment:
I think this is overkill. How about the following refactor:

`return "$window_aggregate" + std::to_string(index);`